bootstrap/viaproxy/src/main/java/org/geysermc/geyser/platform/viaproxy/GeyserViaProxyPlugin.java

```diff
@@ -120,6 +122,7 @@ public void onGeyserInitialize() {
         }
 
         this.geyser = GeyserImpl.load(PlatformType.VIAPROXY, this);
+        GeyserImpl.getInstance().eventBus().register(this, new GeyserServerTransferListener());
```

Comment:
should've made that clearer, now you can just use the geyser variable that was just created
```suggestion
        geyser.eventBus().register(this, new GeyserServerTransferListener());
```